src/bin/super/cmd.rs

```diff
@@ -146,7 +146,7 @@ pub struct AccessNodesInfo {
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct CreateEc2InstanceInfo {
-    pub name: String,
+    pub name: Option<String>,
```

Comment:
I think this will break some other paths of the code. Like where we are creating reserved swarms

Reply:
I've added the ? for the name and I got it compiled, so should be ok no?